tilequeue/process.py

```diff
@@ -5,15 +5,30 @@
 from tilequeue.tile import coord_to_mercator_bounds
 from tilequeue.tile import pad_bounds_for_zoom
 from tilequeue.tile import tolerance_for_zoom
-from tilequeue.transform import mercator_point_to_wgs84
+from tilequeue.transform import mercator_point_to_lnglat
 from tilequeue.transform import transform_feature_layers_shape
 from tilequeue.transform import calculate_padded_bounds
 from TileStache.Config import loadClassPath
```

Comment:
If we've removed TileStache from the dependencies in `setup.py`, do we also need to drop this import and have a local copy of `loadClassPath`?